recipes-core/swupdate/swupdate.bbappend

```diff
@@ -9,16 +9,11 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
 SRC_URI += "file://swupdate.install \
-            file://swupdate.handler.uboot.ini \
-            file://acknowledge_update.sh"
+            file://acknowledge_update.sh \
+            file://swupdate.handler.u-boot.ini"
```

Comment:
Can we name the file as it should be from the begining? Avoids back and forth in the series.